data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -276,6 +276,33 @@
     "mapping_inequality_df"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fceb3136",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load CDC SVI Index\n",
+    "RELEVANT_CDC_SVI_INDEX_COLUMNS = [\n",
```

Comment:
Because `other_census_tract_fields_to_keep` is deprecated, you don't actually need this either.

Reply:
Okay